src/nccl_ofi_net.c

```diff
@@ -119,7 +119,7 @@ static const char *nccl_ofi_req_state_str(nccl_ofi_req_state_t state)
 	}
 }
 
-static const char *nccl_ofi_req_direction_str(nccl_ofi_req_state_t state)
```

Comment:
nit: Could you also change the name of the variable to direction?

Reply:
Done